src/file-uploader/file-uploader.component.ts

```diff
@@ -155,12 +189,11 @@ export class FileUploader {
 		}
 	}
 
-	onFilesAdded() {
-		const files = this.fileInput.nativeElement.files;
+	onFilesAdded(fileList) {
 		if (!this.multiple) {
 			this.files.clear();
 		}
-		for (let file of files) {
+		for (let file of fileList) {
```

Comment:
Can we just use `this.fileList` directly here instead of passing it through the template first?